training/README.md

```diff
@@ -0,0 +1,160 @@
+## Train the Model with Your Own Data
+
+This document provides instructions to train the model on Watson Machine Learning, an offering of IBM Cloud. The instructions in this document assume that you already have an IBM Cloud account. If not, please create an [IBM Cloud](https://ibm.biz/Bdz2XM) account. 
+
+- [Prepare Data for Training](#prepare-data-for-training)
+- [Train the Model](#train-the-model)
+- [Rebuild the Model Serving Microservice](#rebuild-the-model-serving-microservice)
```

Comment:
model-serving (link text should match section heading)

Reply:
Will make Patrick's change, but will leave the conversation unresolved until get some feedback on this